content/Assets/Scripts/components/addClassOnScroll/index.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Applies a class when page height has been scrolled
+ */
+const SELECTOR = '.js-add-class-on-scroll';
+const defaultCSS = 'page-scrolled';
+let pageHeight = window.innerHeight;
+const $scrollElements = document.querySelectorAll(SELECTOR);
```

Comment:
I don't think the selector should happen here as this point could be fired before the page has fully loaded. Ideally it would be within `addClassOnScroll` because that is executed after the page has fully loaded.